sdk/monitor/Azure.Monitor.Ingestion/tests/MonitorIngestionTestEnvironment.cs

```diff
@@ -7,18 +7,18 @@
 
 namespace Azure.Monitor.Ingestion.Tests
 {
-    public class IngestionClientTestEnvironment : TestEnvironment
+    public class MonitorIngestionTestEnvironment : TestEnvironment
     {
-        public string DCRImmutableId => GetRecordedVariable("AZURE_MONITOR_INGESTION_LOGS_DCR_RULE_ID");
+        public string DCRImmutableId => GetRecordedVariable("AZURE_MONITOR_INGESTION_LOGS_DCR_RULE_ID", options => options.IsSecret());
```

Comment:
Is this actually a secret? Can someone do something with this ID without corresponding access keys or Identity?